tests/monitoring/monitoring.go

```diff
@@ -640,8 +645,70 @@ var _ = Describe("[Serial][sig-monitoring]Prometheus Alerts", Serial, decorators
 			libwait.WaitForVirtualMachineToDisappearWithTimeout(vmi, 240)
 		})
 	})
+
+	Context("Warning alerts", func() {
+		alerts := []string{
+			"KubeVirtComponentExceedsRequestedCPU",
+			"KubeVirtComponentExceedsRequestedMemory",
+		}
+
+		BeforeEach(func() {
+			virtClient, err = kubecli.GetKubevirtClient()
```

Comment:
There was a refactoring around getting the kv client, please see [here](https://github.com/kubevirt/kubevirt/pull/8964). You won't need to check `err` anymore. I think that 
you will probably need to do a rebase as well.

Reply:
yes, after rebasing I had to remove those